apps/emqx_rule_engine/src/emqx_rule_engine_schema.erl

```diff
@@ -295,6 +295,14 @@ rule_engine_settings() ->
                     deprecated => {since, "v5.0.22"},
                     importance => ?IMPORTANCE_HIDDEN
                 }
+            )},
+        {sql_actions_undefined_vars_as_null,
```

Comment:
This impacts ALL rules.
Maybe should be a config for each rule.

Reply:
We do want it impacts to all `actions` that are able to insert nullable values into databases.

Reply:
Renamed to `db_actions_undefined_vars_as_null`.

Reply:
refactored to add dedicate options for each action